api/src/main/java/org/pmiops/workbench/monitoring/views/OpenCensusView.java

```diff
@@ -22,12 +22,12 @@
  * we may wish to support other metrics backends, and don't want to depend on Stackdriver concepts
  * or implementation details.
  */
-public interface OpenCensusStatsViewInfo {
+public interface OpenCensusView {
```

Comment:
Naming nitpick: I'd slightly lean towards this being name more in alignment with the classes that implement the interface, e.g. MetricBase or Metric. Now that we're using some non-opencensus terminology for classes in this package (which I think is a good thing – GaugeMetric and EventMetric are exceedingly clear), it may be best to align this interface with the concrete classes.

(Related note, it may be worth updating the javadoc comment re: naming, since it seems slightly out of date now.)

Reply:
you read my mind.